src/videostreamindex/DefaultVideoStreamIndex.ts

```diff
@@ -355,6 +355,24 @@ export default class DefaultVideoStreamIndex implements VideoStreamIndex {
     return this.subscribeSsrcToStreamMap.get(ssrcId);
   }
 
+  overrideStreamIdMappings(previous: number, current: number): void {
+    if (this.subscribeTrackToStreamMap) {
+      for (const [track, streamId] of this.subscribeTrackToStreamMap.entries()) {
+        if (previous === streamId) {
+          this.subscribeTrackToStreamMap.set(track, current);
+        }
+      }
```

Comment:
Can there be multiple entries in the map that share a stream ID? If not, `break` out of the loop.